python/lsst/obs/decam/decamMapper.py

```diff
@@ -233,7 +238,11 @@ def std_raw(self, item, dataId):
 
     def std_dark(self, item, dataId):
         exp = afwImage.makeExposure(afwImage.makeMaskedImage(item))
-        exp.getCalib().setExptime(1.0)
+        rawPath = self.map_raw(dataId).getLocations()[0]
+        headerPath = re.sub(r'[\[](\d+)[\]]$', "[0]", rawPath)
+        md0 = afwImage.readMetadata(headerPath)
```

Comment:
@hchiang2 should comment here about whether the darks are MEFs or not.  I suspect they are not any more.